src/Scripting/Core/ScriptOptions.cs

```diff
@@ -19,11 +21,51 @@ public sealed class ScriptOptions
     {
         public static ScriptOptions Default { get; } = new ScriptOptions(
             filePath: "",
-            references: ImmutableArray<MetadataReference>.Empty,
+            references: GetDefaultMetadataReferences(),
             namespaces: ImmutableArray<string>.Empty,
             metadataResolver: RuntimeMetadataReferenceResolver.Default,
             sourceResolver: SourceFileResolver.Default);
 
+        private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
+        {
+            string corlibDir = RuntimeMetadataReferenceResolver.DefaultCorLibDirectoryOpt;
+
+            if (corlibDir == null)
+            {
+                return ImmutableArray<MetadataReference>.Empty;
+            }
+
+            // Provide similar surface to mscorlib (netstandard 2.0).
+            // These references are resolved lazily. Keep in sync with list in core csi.rsp.
+            var files = new[]
```

Comment:
BTW, csi.exe as well as the InteractiveWindow do add references to System, System.Core, Microsoft.CSharp (to support dynamic), and System.ValueTuple in its .rsp file by default for convenience.
.NET Core version adds corresponding assemblies as well. See https://github.com/dotnet/roslyn/pull/15550/files#diff-b119f0714b2d877b4e719a887841ca6f.